extension/src/setup/studio.ts

```diff
@@ -169,15 +170,27 @@ export class Studio extends Disposable {
     return accessToken
   }
 
-  private async requestStudioToken(deviceCode: string, tokenUri: string) {
-    const token = await this.fetchStudioToken(deviceCode, tokenUri)
-    const cwd = this.getCwd()
+  private requestStudioToken(deviceCode: string, tokenUri: string) {
+    return Toast.showProgress('Connecting to Studio', async progress => {
```

Comment:
Originally, I was going to update our Studio section with a loading message but, after a 1v1 with Stephanie, decided on using a progress Toast instead since:

* The process is practically near instant for someone with a fast computer, it was just looking really slow from my end since my computer is on the slower side. 
*  A Toast is less code to do compared to needing to send a message back and forth to the webview.